### DISCUSSION TITLE
The whole site is really confusing

### DISCUSSION THREAD

**Discussion by ghost** [community/community#140821] [New to GitHub]
I don't know what anything does and nothing makes sense

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @isaas7 ! We're happy you're here. You are more likely to get a useful response if you are specific with what you need help with and are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
